File: JavaScript/ifStatements/taxBracket.js
const readline = require('readline');
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
})

function taxBracket(income) {
    if (income <= 10000) {
        return '10%';
    } else if (income <= 50000) {
        return '20%';
    } else {
        return '30%';
    }
}

rl.question("Please enter you annual income: ", (input) => {
    const annual = parseFloat(input);

    if (isNaN(annual) || annual <0) {
        console.log("Invalid Entry");
    } else {
        const bracket = taxBracket(annual);
        console.log(`With your annual income of $${annual.toFixed(2)}, you fall into the ${bracket} tax bracket.`);
    }
    rl.close();
});

File: ifStat/src/taxBracket.java
import java.util.Scanner;

public class taxBracket {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the amount: ");
        double amount = sc.nextDouble();

        if (amount < 10000) {
            System.out.println("The tax bracket is 10%");
        } else if (amount < 20000) {
            System.out.println("The tax bracket is 20%");
        } else {
            System.out.println("The tax bracket is 30%");
        }
        sc.close();
    }
}
